lib/trento_web/openapi/v1/schema/activity_log.ex

```diff
@@ -0,0 +1,34 @@
+defmodule TrentoWeb.OpenApi.V1.Schema.ActivityLog do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "ActivityLog",
+    description: "Activity Log for the current installation.",
+    type: :array,
+    items: %Schema{
+      type: :object,
+      additionalProperties: false,
+      properties: %{
+        type: %Schema{
+          type: :string,
+          description: "Type of Activity Log entry."
+        },
+        actor: %Schema{
+          type: :string,
+          description: "Actor causing an Activity Log entry. E.g. System or a specific user."
+        },
+        metadata: %Schema{
+          type: :object
+        },
+        inserted_at: %Schema{
```

Comment:
Now that I see this again: what about exposing it as `occurred_on` or something similar, rather than a very database oriented `inserted_at`?

Reply:
In my understanding, `occured_on` will need to be a application-code generated timestamp in a separate field, not a DB auto-generated one. Maybe that can be a followup/later task ?

Reply:
I was thinking that what we `inserted_at` becomes `occurred_on` in the view layer.
Not a biggie tho. Up to you.